src/pages/myAdmin/releaseNotes.tsx

```diff
@@ -0,0 +1,868 @@
+      </li>
+      <li>
+        The <strong>CreateExternalVendorDeviceBulkAsync</strong> method allows Marketplace Partners to register devices using serial numbers, as well as the option to include IMEI and SIM numbers, so they show up in MyAdmin.
+      </li>
+      <li>
+        To use this method, you must have the <code className="small-code-sample">DeviceRegister-Bulk</code> role.
+      </li>
+    </ul>
+  </div>
+);
+
+const updateSeptember192022: ReactNode = (
+  <div className="paragraph">
+    <h2>ApiDeviceContract</h2>
+    <ul>
+      <li>
+        <strong>ApiDeviceContract</strong> has been updated with a boolean value for the <code className="small-code-sample">isAutoActivated</code> parameter to indicate whether a device was auto-activated by Geotab using True or False.
+      </li>
+      <li>
+        The True label indicates the device was auto-activated by Geotab.
+      </li>
+    </ul>
+    <h2>AccountsFromSerialsAsync</h2>
+    <ul>
+      <li>
+        A new API, <strong>GetAccountsFromSerialsAsync</strong>, has been added to allow our Partners with multiple accounts identify the account associated with a specified device, and facilitates the use of other MyAdmin APIs that require account information.
+      </li>
+    </ul>
+    <h2>OnlineOrderStatus</h2>
+    <ul>
+      <li>
+        A new parameter, <code className="small-code-sample">poExactMatch</code>, has been added to <strong>GetOnlineOrderStatus</strong> to allow youto search for orders that match a specific PO number.
+      </li>
+      <li>
+        To search for orders that match an exact PO number, use double quotation marks.
+      </li>
+    </ul>
+    <h2>End of Life — MyAdmin APIs for Third-party Device Data Ingestion</h2>
+    <ul>
+      <li>
+        On September 30, 2022, the MyAdmin APIs for third-party device data ingestion into MyGeotab will be permanently discontinued. After this date, MyGeotab will be unable to receive data from any devices that have not migrated to DIG.
+      </li>
+      <li>
+        For more information, refer to the{" "}
+        <a
+          href="https://docs.google.com/document/d/1hJj2CEMJlgbXHjF-it8Gey8b5_EnUDCqqhS8jRtxGyw/edit#heading=h.dzk3awl3nlud"
+          aria-label="Migrating to DIG from MyAdmin API/SDK guide"
+          target="_blank"
+          rel="noopener noreferrer">
+          Migrating to DIG from MyAdmin API/SDK guide
+        </a>
+        .
+      </li>
+      <li>
+        If you're a new Partner with a standalone telematics device, refer to the{" "}
+        <a
+          href="https://geotab.github.io/sdk/software/guides/custom-telematics-devices/"
+          aria-label="Using Custom Telematics Devices"
+          target="_blank"
+          rel="noopener noreferrer">
+          Using Custom Telematics Devices
+        </a>{" "}
```

Comment:
Need a TODO here so that we can validate the link eventually since this is a page in the SDK site.